user_manual/changes/U624-010.yaml

```diff
@@ -0,0 +1,7 @@
+type: bugfix
+title: Fix support of ``'Base`` attribute
+description: |
+    This patch fixes a bug where using the ``'Base`` attribute as the prefix
```

Comment:
*adds support for the base attribute ? We didn't have it before

Reply:
No we had some support for it!